### DISCUSSION TITLE
Markor Markdown extensions / Admonition / Boxed content warnings

### DISCUSSION THREAD

**Discussion by blogdemoi** [gsantner/markor#1746] [Q&A] [answered]
Hello,

I just read a blog post about Markor https://gsantner.net/blog/2022/05/14/markor-v2.9-snippets-zimwiki-math-daynight-privacy-markdown-todotxt.html
and I am wondering how to add the Admonition Extension to Markor.

Thank you.

**Comment by gsantner** [comment, marked as answer]
The Markdown Admonition extension is enabled and you don't need to do anything here. Just use the correct syntax as you see below.  
As of possibilities, everything Admonition offers is shown in the example including the category names.

In Markor main screen at the filemanager, press the big red + button to create a new file.  
At the new file dialog you can find predefined templates, among that is the Markdown reference.
And that reference contains exactly what is shown in the blog post screenshot.  


![grafik](https://user-images.githubusercontent.com/6735650/176217689-caaffc05-1345-47fc-8da6-b46b3d8328ab.png)

**Reply by blogdemoi** [reply]
Thank you.
